components/OurTeam/Partners.tsx

```diff
@@ -9,32 +10,42 @@ import {
 import { PageTitle, PageDescr } from "./CommonComponents"
 
 export const OurPartners = () => {
+  const { t } = useTranslation("common")
+
+  // linter does not like literal strings
+
+  const untranslatedProperTitle1 = "NuLawLab"
```

Comment:
OOC, why can't these be in the text file? We do ultimately control the translation result.

Reply:
they can, I wasn't sure if translation would handle proper names well or not

but if they do, I can totally simplify this.  Just chalk this up to an overabundance of caution.

Reply:
Definitely a good callout - not sure how well the *automated* translations will handle these, but this can definitely be handled in the manual validation pass (which we'd need anyway).